test/test-ipc.c

```diff
@@ -838,10 +838,10 @@ static unsigned int write_until_data_queued() {
                  closed_handle_large_write_cb);
     ASSERT(r == 0);
     i++;
-  } while (((uv_stream_t*)&channel)->write_queue_size == 0 &&
+  } while ((&channel)->write_queue_size == 0 &&
            i < ARRAY_SIZE(write_reqs));
 
-  return ((uv_stream_t*)&channel)->write_queue_size;
+  return (&channel)->write_queue_size;
```

Comment:
I think these can be just `channel.write_queue_size` then?

Reply:
Thanks for your feedback. I have had it resolved.